src/main/java/com/schibsted/security/artishock/cli/view/Commands.java

```diff
@@ -226,14 +226,17 @@ public static class NotClaimed extends BaseCommand {
     @io.airlift.airline.Option(name = LOCAL_NAME, description = LOCAL_DESCRIPTION, required = true)
     public String local;
 
+    @io.airlift.airline.Option(name = "--excluded", description = "File containing excluded packages")
```

Comment:
To align with existing style, it should probably have `, required = false` added here.

There already is a description with exactly the same text [here](https://github.com/schibsted/artishock/blob/main/src/main/java/com/schibsted/security/artishock/cli/view/Commands.java#L169), I would pull it out in a constant as it's done for the other arguments that are valid for multiple commands.

Reply:
`required = false` is redundant and results in a intellij lint complaint, so I removed it instead.

The description text is now moved out to a constant.